source/working_remotely.rst

```diff
@@ -0,0 +1,60 @@
+################
+Working Remotely
+################
+
+Some nao robots are always turned on in the Robotics Lab for working remotely via the Kodos server.
+
+*****************************
+Connecting to Kodos using SSH
+*****************************
+
+First, your computer should be connected to Kodos using SSH.
+
+.. note::
+    For creating your account on Kodos, please :ref:`contact` us.
+
+Run the following command in a terminal of your computer to connect to Kodos.
+
+.. code-block:: bash
+
+   $ ssh -L 5903:localhost:5903 username@kodos.cse.unsw.edu.au
+
+***************************
+Running VNC Server in Kodos
+***************************
+
+Run :code:`vncserver -geometry <dim>` in Kodos such as
+
+.. code-block:: bash
+
+   $ vncserver -geometry 1920x893
```

Comment:
since this will run in foreground, i suggest you tell people either a) how to background it; or b) to open a new terminal+ssh

Reply:
`vncserver -geometry 1920x893` created a background process and the cursor appears for the next command. So, & in the end, would not be required. 

% I used a terminal of MacBook and connected to Kodos using SSH. Then, ran `vncserver -geometry 1920x893`. 

Could you confirm whether this is still a problem?

Reply:
sorry, you are right.  i don't know why i have `&` in my command history